Sources/Basic/OutputByteStream.swift

```diff
@@ -59,13 +59,13 @@ public class OutputByteStream: OutputStream {
     ///
     /// This method implicitly flushes the stream.
     public var bytes: ByteString {
-        flush()
+        try? flush()
```

Comment:
To be a little more clear, the main reason this is desired is so that the file-backed versions can write very large files without buffering all the contents in memory.